src/styles/nested-list.scss

```diff
@@ -5,378 +5,253 @@ $block: #{$fd-namespace}-nested-list;
 $button: #{$fd-namespace}-button;
 
 .#{$block} {
-  $fd-nested-list-border-radius: var(--fdNested_List_Border_Corner_Radius);
-
-  @mixin fd-nested-level-padding($paddingLeft, $paddingRight) {
-    .#{$block}__link,
-    .#{$block}__content {
-      padding-left: $paddingLeft;
-      padding-right: $paddingRight;
-      border-bottom: none;
-
-      @include fd-selected() {
-        border-bottom: var(--sapList_BorderWidth) solid var(--sapList_SelectionBorderColor);
-      }
-
-      &.has-child {
-        padding-left: 0;
-        padding-right: 0;
-      }
-    }
-
-    .#{$block}__content {
-      &.has-child {
-        > .#{$block}__title {
-          padding-left: $paddingLeft;
-          padding-right: $paddingRight;
-        }
-      }
-    }
-  }
-
-  @mixin fd-nested-list-popover-border-top-radius {
-    border-top-left-radius: $fd-nested-list-border-radius;
-    border-top-right-radius: $fd-nested-list-border-radius;
+  $fd-nested-list-item-height: 2.75rem;
+  $fd-nested-list-item-height-compact: 1.5rem;
+  $fd-nested-list-item-border-radius: 0.5rem;
+  $fd-nested-list-item-spacing: 0.25rem;
+  $fd-nested-list-item-spacing-compact: 0.125rem;
+
+  @mixin fd-nested-list-states-colors($textColor, $backgroundColor) {
+    color: $textColor;
+    background: $backgroundColor;
   }
 
-  @mixin fd-nested-list-popover-border-bottom-radius {
-    border-bottom-left-radius: $fd-nested-list-border-radius;
-    border-bottom-right-radius: $fd-nested-list-border-radius;
+  @mixin fd-nested-list-focus($offset: 0.125rem, $width: 0.125rem) {
+    &:focus,
+    &.is-focus {
+      outline-offset: $offset;
+      outline-width: $width;
+      outline-color: var(--sapContent_FocusColor);
+      outline-style: solid;
+      @content;
+    }
   }
 
   @include fd-reset();
 
-  list-style: none;
   width: 100%;
-  border-bottom: var(--sapList_BorderWidth) solid var(--sapList_BorderColor);
+  list-style: none;
 
-  & [class*="level"] {
-    border-bottom: none;
+  &.level-1 {
```

Comment:
is it correct according to the bem?

Reply:
These class names are being added programmatically in ngx in `nested-list.directive.ts`. Maybe time for a rename but it'd have to be done on both ngx and styles

Reply:
These class names are being added programmatically in ngx in `nested-list.directive.ts`. Maybe time for a rename, it'd have to be done on both ngx and styles

Reply:
we have in fundamental-styles similar cases with the classes `is-open`, `has-child`, etc. 
`level-1`, `level-2`, etc are not modifier classes in the sense of BEM. They don't modify the properties of the element they are attached to, but rather to its child elements.
In BEM we have something like: `fd-nested-list  fd-nested-list--transparent`. This modifies the background of ``fd-nested-list`, but in our case `level-2` would modify the item and the title, which are child and grandchild of the element. In the past we wanted to distinguish such behaviour and that's why classes like `is-open`, `has-child`, `level-*` appeared. I am open to discussion about that. Not a problem at all to rename the classes.

Reply:
Thanks for the explanation, I think just leave them as-is then. If we do want to change, let's make a separate issue and discuss